### DISCUSSION TITLE
Support For fleeq.io Embeds

### DISCUSSION THREAD

**Discussion by spacec0de** [outline/outline#2546] [Ideas]
We use fleeq.io for training videos and tutorials. Unfortunately it's quite different than the existing embed options so there isn't any overlap. 

Instead of normal videos, with fleeq you take screenshots, point, click, highlight, annotate, etc, and then the system weaves the screenshots together to create a live interaction video with chapters and narration.

Anyway, we've found them to be pretty effective for training and would like to be able to embed them in Outline if possible.

**Comment by tommoor** [comment]
Could you provide an example of a fleeq embed?
